pkg/api/routing/routing.go

```diff
@@ -12,16 +12,10 @@ var (
 	}
 )
 
-func Wrap(action interface{}) web.Handler {
+func Wrap(handler func(c *models.ReqContext) response.Response) web.Handler {
```

Comment:
THIS was the whole point of the PR  🔨 .
We finally have all the handlers of the same type and compiler checks for that.

Reply:
Actually, the next PR would be to remove this `routing` package and use router directly instead.